src/django_cloudtask/utils.py

```diff
@@ -1,19 +1,28 @@
+import functools
+import time
+
 import google.auth.transport.requests
 from google.auth import jwt
 from google.oauth2.id_token import _GOOGLE_OAUTH2_CERTS_URL
 from google.oauth2.id_token import _fetch_certs as fetch_certs
 
-certs = None
 
+@functools.lru_cache(1)
+def _get_certs(_ttl_hash):
+    request = google.auth.transport.requests.Request()
+    return fetch_certs(request, _GOOGLE_OAUTH2_CERTS_URL)
```

Comment:
This caching mechanism is very clever (and not super obvious).  I'm guessing because the functools cache doesn't give you the option for a ttl, you've resorted to this instead?

Reply:
Yep that's correct.
I considered using a 3rd party package but didn't want to introduce new dependencies.